src/map.c

```diff
@@ -1428,23 +1438,23 @@ static void map_renderMarkers( double x, double y, double zoom, double r, double
       StarSystem *sys = system_getIndex( i );
 
       /* We only care about marked now. */
-      if (!sys_isFlag(sys, SYSTEM_MARKED | SYSTEM_CMARKED))
+      if (!sys_isFlag(sys, SYSTEM_MARKED | SYSTEM_CMARKED | SYSTEM_PMARKED))
          continue;
 
       /* Get the position. */
       tx = x + sys->pos.x*zoom;
       ty = y + sys->pos.y*zoom;
 
       /* Count markers. */
-      n  = (sys_isFlag(sys, SYSTEM_CMARKED)) ? 1 : 0;
+      n  = (sys_isFlag(sys, SYSTEM_CMARKED | SYSTEM_PMARKED)) ? 1 : 0;
       n += sys->markers_plot;
       n += sys->markers_high;
       n += sys->markers_low;
       n += sys->markers_computer;
 
       /* Draw the markers. */
       j = 0;
-      if (sys_isFlag(sys, SYSTEM_CMARKED)) {
+      if (sys_isFlag(sys, SYSTEM_CMARKED | SYSTEM_PMARKED)) {
```

Comment:
It would probably be better to have the player notes markers de independent of the computer markers and probably have a different shape to be easily identifiable. I guess a truncated rhombus or something like that could work. I can help with the Signed Distance Function Shader if you want.

Reply:
Help would be great .. I know practically nothing about this now.

Reply:
I've separeted things like you've said .. it calls `map_drawMarker( tx, ty, zoom, r, a, n, j, 5 );`

Reply:
Don't you need to increment `n` like?
```C
n  = (sys_isFlag(sys, SYSTEM_CMARKED)) ? 1 : 0;
n  += (sys_isFlag(sys, SYSTEM_PMARKED)) ? 1 : 0;
````

Reply:
Note : marker type 5 colour is temporarely set to &cMarkerNew in map.c:867 .